### ISSUE TITLE
Finetuning Torchvision Models tutorial contains bad link

### ISSUE THREAD

**Issue by neldsman** [pytorch/pytorch.github.io#771]
## 📚 Documentation

In the [load data](https://pytorch.org/tutorials/beginner/finetuning_torchvision_models_tutorial.html#load-data) section there is a link to get additional information on the normalization values for the models outlined in the previous sections which leads to a 404. The link provided is: https://pytorch.org/docs/master/torchvision/models.html which I believe should be updated to https://pytorch.org/vision/stable/models.html
